numpy/core/_add_newdocs.py

```diff
@@ -3998,6 +3998,8 @@
 
     Returns a view of the array with axes transposed.
 
+    Refer to `numpy.transpose` for full documentation.
```

Comment:
Are the docs for `transpose` clearer than these ones?

Reply:
Not necessarly clearer, it has different informations in Notes section.
Although, ndarray.transpose has a better example in how to convert 1-D array into 2D

Reply:
I could change "for full docuentation" to "for more documentation" perhaps

Reply:
I feel like the `ndarray.transpose` has more details, and it's also called by `transpose`, so i changed the way the link between them was being used